front/app/modules/commercial/impact_tracking/index.ts

```diff
@@ -63,6 +66,13 @@ const trackPageView = async (path: string) => {
   // for the same path. This prevents that.
   if (previousPathTracked === path) return;
 
+  // On first homepage load, if we don't enter the platform on a locale,
+  // we set it immediately. Usually, we set it before the page track event
+  // fires, but in some cases our app is too slow.
+  // This then triggers the page view event twice.
+  // With this check, we avoid logging this pageview twice.
+  if (path === '/' || path === '') return;
```

Comment:
Could you also explain in a comment why you check for both `/` and an empty string? I don't get that either.